forest/src/cli/chain_cmd.rs

```diff
@@ -82,17 +85,32 @@ impl ChainCommands {
 
                 let epoch = tipset.unwrap_or(chain_head.epoch());
 
-                let output_path = match output_path {
-                    Some(path) => path.to_owned(),
-                    None => {
-                        let now = OffsetDateTime::now_utc();
-                        format!(
-                            "forest_snapshot_{}-{}-{}_height_{}.car",
-                            now.year(),
-                            now.month(),
-                            now.day(),
-                            epoch,
-                        )
+                let now = OffsetDateTime::now_utc();
+
+                let month_string = format!("{:02}", now.month() as u8);
+                let year = now.year();
+                let day = now.day();
+                let chain_name = cfg.chain.name.clone();
+
+                let default_format =
+                    "forest_snapshot_{chain}_{year}-{month}-{day}_height_{heigth}.car";
+
+                let mut vars = HashMap::new();
+                vars.insert("year".to_string(), year.to_string());
+                vars.insert("month".to_string(), month_string);
+                vars.insert("day".to_string(), day.to_string());
+                vars.insert("chain".to_string(), chain_name);
+                vars.insert("heigth".to_string(), epoch.to_string());
+
+                let output_path_res = match output_path {
+                    Some(path) => strfmt(path, &vars),
+                    None => strfmt(default_format, &vars),
+                };
+
+                let output_path = match output_path_res {
+                    Ok(path) => path,
+                    Err(e) => {
+                        panic!("Unparsable string error: {}", e);
```

Comment:
You can't use `panic!` for a user-facing error message. Use `cli_error_and_die`.